l2geth/core/vm/evm.go

```diff
@@ -203,6 +205,16 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 		statedumper.WriteMessage(caller.Address(), input)
 	}
 
+	if addr == dump.OvmEthAddress {
+		// We need at least 4 bytes + 32 bytes for the recipient address, then
+		// address will be found at bytes 16-36. 0x40c10f19 is the function
+		// selector for mint(address,uint256).
+		if len(input) >= 36 && bytes.Equal(input[:4], common.FromHex("0x40c10f19")) {
```

Comment:
[nit] Can pull the `common.FromHex` out into a constant. Otherwise LGTM.

Reply:
Agreed on this